DEVELOPERS.md

```diff
@@ -8,3 +8,10 @@
 * install composer: `curl "https://getcomposer.org/installer" | php`
 * update `composer.lock`: `php composer.phar update`
 * `rm composer.phar`
+
+## QA
+
+There is an automated [QA tool](https://github.com/codeclimate/qm_qa) that can
```

Comment:
my only question is around linking to this here, since its a private repo, and duplication is public.

Reply:
Boo, good point. I'm inclined to leave it, since QM stuff will be open sourced at some point, and this is an important and valuable step for people adding QM language support. If it comes up as a problem with an open source contributor before we open source it, we'll address it then.